src/App.js

```diff
@@ -0,0 +1,78 @@
+import "./App.css";
+import { useEffect, useState } from "react";
+import { listMealsByFirstLetter, getMealById } from './modules/recipies/recipiesService';
+import { SearchBar } from "./modules/recipies/SearchBar";
+import { RecipeList } from "./modules/recipies/RecipeList";
+
+
+function App() {
+  const [recipes, setRecipes] = useState([]);
+  const [selectedRecipe, setSelectedRecipe] = useState(null);
+
+  const handleSearch = (letter) => {
```

Comment:
When we do a search, we enter a query. So it would be better to rename the parameter ('letter') to 'query'